changelog-entries/180.md

```diff
@@ -0,0 +1 @@
+Disabled execute at start for OpenFOAM8 (issue #179)
```

Comment:
Please add the PR number here, not the issue.

Reply:
Ok, I thought the name of the file would be sufficient.

Reply:
Added it.